stock_picking_completion/models/stock_move_line.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+from odoo.tools.float_utils import float_is_zero
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    completion = fields.Float(compute="_compute_completion", store=True)
```

Comment:
Why do we need this field stored BTW?

Reply:
So we don't have to compute all at once when loading a bunch of those.

Reply:
I.E. when retrieving pickings from `shopfloor_mobile`.
There's no point in this field if it's not stored, actually.